### PR TITLE
[JA] Translate basic 2.3.0

### PR DESCRIPTION
**Pull request by starfish719**
I translated `lessons/ecto/basic` section.
Please review! @elixirschool/japanese

https://github.com/elixirschool/elixirschool/commit/6eee4a0b293bd5ef98d026efc8cc2bb9b5bbb975#diff-a3d9528d5c48b91d7a98fe56337f1bb8
https://github.com/elixirschool/elixirschool/commit/b3ad2555c705f7540d1604e245d64eab98250f3a#diff-a3d9528d5c48b91d7a98fe56337f1bb8

### CODE REVIEW

File: ja/lessons/ecto/basics.md [hunk 1]
```diff
@@ -190,18 +190,18 @@ end
 
 この `Example.Person` は、それが `people` テーブルと関連するものであることをEctoに伝え、stringの `name` とintegerでデフォルトが0の `age` という2つのカラムを持っていることがわかります。
```
Comment (starfish719) [inline ja/lessons/ecto/basics.md]: `Example.Person` 🤔 
maybe`Friends.Person`

I'll try to send pullrequest `en/lessons/ecto/basics.md` next time 👍

Comment (starfish719) [reply ja/lessons/ecto/basics.md]: I sent pullrequest 💪 
https://github.com/elixirschool/elixirschool/pull/1923

**Comment by riseshia** [comment]
LGTM with #1923 
We can bump up 2.2.1 after #1923 merged 😄

**Comment by starfish719** [comment]
version up to 2.3.0 💪 
https://github.com/elixirschool/elixirschool/pull/1923#discussion_r308767465

**Pull request merged**